### DISCUSSION TITLE
Vite template - base path

### DISCUSSION THREAD

**Discussion by Ribosom** [excaliburjs/Excalibur#3273] [Ideas]
Hi,

I noticed an issue with the template-ts-vite: the built game doesn't work properly (when build with `npm run build`), both locally and when uploaded to itch.io, due to vite using absolute path by default.

I found a simple solution by adding a vite.config.js with a default base path:

```js
import { defineConfig } from "vite";

export default defineConfig({
  base: "./",
});
```

After adding this config, the game works perfectly both locally and on itch.io.

Would it make sense to include this config in the template by default?

PS: Thanks for the engine. I really appreciate it!

**Comment by eonarheim** [comment]
@Ribosom Totally, I think this would be a great idea to add this to the template by default. 

Apologies I'm catching up on notifications after the holiday

**Reply by eonarheim** [reply]
@Ribosom Thanks again for the kind words!

I've made the change to the default template https://github.com/excaliburjs/template-ts-vite
